classifiers/reference_quality/special_character_classifier/README.md

```diff
@@ -0,0 +1 @@
+The purpose of this brick is to identify if there are any unusual characters in the given text. This function can be useful for text preprocessing tasks, especially for checking reference material in RAG (Retrieval Augmented Generation) use cases where you want to filter out text that contains unusual or unexpected characters.
```

Comment:
do we have a common pattern here? "The purpose of this brick is"  sounds different to other bricks descriptions

Reply:
I can of course change it if it's sound weird. There is no given pattern for the style of the README, other than that it should state a description of what the brick does, provide valuable information etc.